grc/gui_qt/components/canvas/block.py

```diff
@@ -117,7 +130,11 @@ def create_shapes_and_labels(self):
         else:
             for key, item in self.core.params.items():
                 value = item.value
-                if (value is not None and item.hide == "none") or (item.dtype == 'id' and self.force_show_id):
+                if (
+                    (value is not None and item.hide == "none") or
+                    (item.dtype == "id" and self.force_show_id) or
+                    (item.dtype == "id" and self.core.key == "options")
```

Comment:
This is one of two changes in this PR. The rest is all whitespace, comments, formatting, ....